LA.UM.9.12.r1.xml

```diff
@@ -8,6 +8,7 @@
 <project path="kernel/sony/msm-4.19/kernel/techpack/camera" name="kernel-techpack-camera" groups="device" remote="sony" revision="aosp/LA.UM.9.12.r1" />
 <project path="kernel/sony/msm-4.19/kernel/techpack/data-kernel" name="kernel-techpack-data-kernel" groups="device" remote="sony" revision="aosp/LA.UM.9.12.r1" />
 <project path="kernel/sony/msm-4.19/kernel/techpack/display" name="kernel-techpack-display-driver" groups="device" remote="sony" revision="aosp/LA.UM.9.12.r1" />
+<project path="kernel/sony/msm-4.19/kernel/techpack/video" name="vendor-qcom-opensource-video-driver" groups="device" remote="sony" revision="aosp/LA.UM.9.12.r1" />
```

Comment:
Please change the project name to
kernel-techpack-video

Reply:
@jerpelea Both repositories are still live, can you please delete one of them entirely?

https://github.com/sonyxperiadev/kernel-techpack-video
https://github.com/sonyxperiadev/vendor-qcom-opensource-video-driver

The former has an extra `.gitignore` on the 9.14 branch courtesy of @konradybcio, both repos are identical in branches and HEAD otherwise.

Reply:
The first one to keep would make more sense to keep them in line with the rest.